ToolBox/TickAggregator.cs

```diff
@@ -44,7 +44,13 @@ protected TickAggregator(Resolution resolution, TickType tickType)
         /// </summary>
         public List<BaseData> Flush()
         {
-            return new List<BaseData>(Consolidated) { Consolidator.WorkingData as BaseData };
+            var data = new List<BaseData>(Consolidated);
+            if (Consolidator.WorkingData == null)
```

Comment:
This bit is odd; possible just need to invert the check to `!=`

Reply:
Yes, this is certainly odd -- thanks for spotting -- I've performed a `--fixup` on the `HEAD` commit of this PR fixing the offending conditional.